lib/galaxy/dependencies/conditional-requirements.txt

```diff
@@ -28,3 +28,7 @@ watchdog
 python-gitlab
 pygithub
 influxdb
+
+# Deep learning packages for tool recommendation
+keras==2.2.4
+tensorflow==1.15.0
```

Comment:
@anuprulez can we use the 1.15.2 version here?

Reply:
@bgruening I can but it will not make much difference